Automattic/wp-calypso
# Sharing: Remove SharingConnectionsData component

This pull request seeks to remove the `<SharingConnectionsData />` component. It was only ever used once, in the context of the post editor for managing connections data, though this has since been replaced with [`<EditorSharingContainer />`](https://github.com/Automattic/wp-calypso/blob/master/client/post-editor/editor-sharing/container.jsx) in #338.

__Testing instructions:__

Verify that I haven't missed any references to the component, and that the [post editor](http://calypso.localhost:3000/post) continues to load and display Publicize connections in the Sharing sidebar accordion.
